packages/gatsby/gatsby-browser.js

```diff
@@ -25,8 +25,6 @@ exports.onClientEntry = function (_, pluginParams) {
   }
 
   Sentry.init({
-    // eslint-disable-next-line no-undef
-    release: __SENTRY_RELEASE__,
```

Comment:
m: Let's just check that removing this doesn't change/break anything for Gatsby

Reply:
Tested it in a gatsby app and it still works!